stl/debugger/STL.natvis

```diff
@@ -1443,15 +1443,26 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
       </Expand>
   </Type>
 
+  <!-- VC 2015 - must have higher priority -->
+  <Type Name="stdext::hash_map&lt;*&gt;" Priority="Medium">
+      <AlternativeType Name="stdext::hash_multimap&lt;*&gt;" />
+      <AlternativeType Name="stdext::hash_set&lt;*&gt;" />
+      <AlternativeType Name="stdext::hash_multiset&lt;*&gt;" />
+      <DisplayString>{_List}</DisplayString>
+      <Expand>

```

Comment:
No change requested: In theory, `comp` could be a function pointer worth visualizing. However, it seems likely that it would have to be paired with an ADL-customized `hash_value` and that seems quite rare (given the poorly-documented nature of this non-Standard extension). I'm okay with this as-is.